README.md

```diff
@@ -1 +1,237 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Kanban Board <a name="kanban-board"></a>
+Hi there, we are a five-member team, [@Thinus01](https://github.com/Thinus01), [@raihan2bd](https://github.com/raihan2bd) , [@edubew](https://github.com/edubew) , [@MichealKith](https://github.com/MICHAELKITH) , [@lawrahkonwea](https://github.com/lawrahkonwea).
+
+We are dividing the project between [front-end](https://github.com/Thinus01/Resort_Booking_Front-end) and [back-end](https://github.com/Thinus01/Resort_Booking_Back-end)
+We also created two kanban boards together for the projects
+Front-end: https://github.com/users/Thinus01/projects/11
+Back-end: https://github.com/users/Thinus01/projects/10
+
+![Screenshot 2023-07-12 163157](https://github.com/Thinus01/Resort_Booking_Back-end/assets/78565848/51806d7f-86ff-4ace-8b17-4528f9846cc6)
+![Screenshot 2023-07-12 163226](https://github.com/Thinus01/Resort_Booking_Back-end/assets/78565848/b058c621-63ff-4bbe-90c6-72ee7a1ef5e6)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a> 
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- First of all to see this project's graphical interface make sure you run the [front-end](https://github.com/Thinus01/Resort_Booking_Front-end) part
+- Before running this repo on your local machine make sure you have installed <a name="install"></a> [Ruby](https://www.ruby-lang.org/), [Rails](https://rubyonrails.org/), and [PostgreSQL](https://www.postgresql.org/)
+
+### Setup
+
+- Clone this repository to your desired folder:
+
+```sh
+  cd your-folder
+  https://github.com/Thinus01/Resort_Booking_Back-end
+```
+
+- Before running the project please make sure you create a `.env` file to your project root directory and add `DATABASE_USER_NAME`, `DATABASE_PASSWORD`, and `JWT_SECRET_KEY` environment variables to the file. For example:
+```
+DATABASE_USER_NAME=place_your_postgres_database_username
+DATABASE_PASSWORD=place_your_database_password
+
+JWT_SECRET_KEY=place_your_jwt_secret_key
+```
+![back-end-env](https://github.com/Thinus01/Resort_Booking_Back-end/assets/35267447/3fd60673-53a2-4e6c-a386-ff894546c850)
+
+- After that make sure you change cors. To change the cors open the project with your favorite editor then click config folder then click initializers folder and open cors.rb file and after that change the `origins` to your front-end URL to solve the cross-origins request error. For example:
+```
+origins 'http://localhost:3000'
+```
+![cors](https://github.com/Thinus01/Resort_Booking_Back-end/assets/35267447/44eb387a-c8d5-40c8-b910-860ed9bf0195)
+
+
+### Install
+
+Install this project with:
+
+- Install the required gems with:
+
+```sh
+bundle install
+npm install
+```
+
+### Database
+
+- Create the databases and run migrations with:
+
+```sh
+rails db:create
+rails db:migrate
+rails db:seed
+```
```

Comment:
- [ ] As mentioned in the last review, it is impossible to create the database as a new user:

![image](https://github.com/Thinus01/Resort_Booking_Back-end/assets/57421823/aae796f8-13d7-4d94-9976-57b66c17f4fa)

After extensive investigating, I was finally able to create the database by deleting these 2 lines in `spec/factories/booking.rb`:

![image](https://github.com/Thinus01/Resort_Booking_Back-end/assets/57421823/b019a0af-a3c9-40dd-a962-194ae23d63e4)

What is happening, is that `resort = Resort.first` is being evaluated at the time of creating the database, which of course is impossible since the database does not exist yet.

On your end, this is working because your database already exists, but for a new user (like external reviewers, the final evaluator, or even potential employers) this bug won't allow them to create the database at all.

If you want to reproduce this bug on your end and test its existence, drop the database with `rails db:drop` and try to create the database again.

Kindly, fix this bug so your project can be setup and run correctly by new users.  👍 

_note: I'll continue the review by deleting those 2 lines in `booking.rb`_

Reply:
@Meltrust Thank you for your invaluable suggestions! Your help was crucial in resolving the issues. I have fixed everything, and the project is now working properly. Thanks again! 👍